frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard.tsx

```diff
@@ -54,7 +54,7 @@ const makeOCSRequest = (state: InternalClusterState): Promise<StorageClusterKind
   );
   const promises: Promise<K8sResourceKind>[] = [...labelNodes(nodes), labelOCSNamespace()];
   if (encryption.advanced && kms.hasHandled) {
-    promises.push(...createKmsResources(kms));
+    promises.push(...createClusterKmsResources(kms));
```

Comment:
Please update the same here in odf flow.
https://github.com/openshift/console/blob/1a4097151e207ab464e2cefb45f2afb5ed984098/frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx#L145-L149

Here is the reference to the state you might need:

https://github.com/openshift/console/blob/68681795168b81e0a7a1de02a886d687aaac7c06/frontend/packages/ceph-storage-plugin/src/components/create-storage-system/reducer.ts#L125-L131

Reply:
I am not quite sure if get it right, but, are you talking about this ??
```
        const promises: Promise<any>[] = [];
        if (state.capacityAndNodes.nodes)
          promises.push(...labelNodes(state.capacityAndNodes.nodes));
        promises.push(labelOCSNamespace());
        if (state.securityAndNetwork.encryption.advanced && state.securityAndNetwork.kms.hasHandled)
          promises.push(...createKmsResources(state.securityAndNetwork.kms));
        const requests = payloads.map(({ model, payload }) => k8sCreate(model as K8sKind, payload));
        await Promise.all([promises, ...requests]);
  ```
  we will add this here: https://github.com/openshift/console/blob/1a4097151e207ab464e2cefb45f2afb5ed984098/frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx#L178

Reply:
Will raise a separate PR for this !!